test/python/test_identifiers.py

```diff
@@ -301,8 +301,7 @@ def test_add_circuit_noname(self):
         result = q_program.execute(backend=backend, shots=shots, seed=78)
         counts = result.get_counts(new_circuit.name)
         target = {'00': shots / 2, '01': shots / 2}
-        threshold = 0.025 * shots
-        self.assertDictAlmostEqual(counts, target, threshold)
+        self.assertDictAlmostEqual(counts, target)
```

Comment:
Rather than change the default value of `assertDictAlmostEqual` use:
```
threshold = 0.04 * shots
self.assertDictAlmostEqual(counts, target, threshold)
```